examples/all_in_one/slo-exporter/conf/slo_exporter.yaml

```diff
@@ -0,0 +1,48 @@
+webServerListenAddress: "0.0.0.0:8001"
+maximumGracefulShutdownDuration: "10s"
+afterPipelineShutdownDelay: "1s"
+
+pipeline: ["tailer", "relabel", "eventKeyGenerator", "dynamicClassifier", "sloEventProducer", "prometheusExporter"]
+
+modules:
+
+  tailer:
+    tailedFile: "/logs/access_log"
+    follow: true
+    reopen: true
+    positionFile: ""
+    positionPersistenceInterval: "2s"
+    loglineParseRegexp: '^(?P<ip>[A-Fa-f0-9.:]{4,50}) \S+ \S+ \[(?P<time>.*?)\] "\s*(?P<httpMethod>GET|POST|HEAD|UPDATE|DELETE|PUT|CONNECT|OPTIONS|TRACE|PATCH)\s+(?P<httpPath>[^\s]+)\s+(?P<protocolVersion>[^\s]+)?\s*" (?P<statusCode>\d+) \d+ "(?P<referer>.*?)" uag="(?P<userAgent>[^"]+)" "[^"]+" ua="[^"]+" rt="(?P<requestDuration>\d+(\.\d+)??)".+ignore-slo="(?P<ignoreSloHeader>[^"]*)" slo-domain="(?P<sloDomain>[^"]*)" slo-app="(?P<sloApp>[^"]*)" slo-class="(?P<sloClass>[^"]*)" slo-endpoint="(?P<sloEndpoint>[^"]*)" slo-result="(?P<sloResult>[^"]*)"'
```

Comment:
I'd either add requests which send HTTP headers with the `slo-*` headers or drop them from the log format and the regexp so there is only configuration which is really used.

Reply:
Log format refactored within https://github.com/seznam/slo-exporter/pull/4/commits/405c273a358be5e2da73147144ac90e713640fe3